__tests__/app.test.js

```diff
@@ -189,6 +201,24 @@ describe("/api/articles", () => {
           );
         });
     });
+
+    test("GET 404: responds with 404 for non-existent topic", () => {
+      return request(app)
+        .get("/api/articles?topic=non_existent_topic")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("404 - Not Found: topic  not found");
+        });
+    });
+
+    test("GET 404: responds with 400 if topic value missing", () => {
+      return request(app)
+        .get("/api/articles?topic=")
```

Comment:
This should be a 200 :)

Reply:
Oh, right! I missed this - thank you very much for pointing this out. :)